docs/source/release/v4.3.0/indirect_geometry.rst

```diff
@@ -49,5 +49,7 @@ BugFixes
 - The Abins file parser no longer fails to read data from non-periodic vibration calculations performed with CRYSTAL17.
 - :ref:`ApplyPaalmanPingsCorrection <algm-ApplyPaalmanPingsCorrection>` will now run also for fixed window scan reduced data and will not crash on workspace groups.
 - Fixed a bug crashing the ``Indirect ILL Data Reduction GUI`` when Run is clicked.
+- Indirect ILL reductions for BATS and Doppler QENS modes will now flag the outputs as distributions if the data is normalised by the monitor.
```

Comment:
Could you also write a test for this change?

Reply:
This is captured by the system test, that's why the references are updated.

Reply:
OK